packages/common/src/ee/AiAgent/schemas/tools/toolTableVizArgs.ts

```diff
@@ -20,7 +20,7 @@ export const toolTableVizArgsSchema = createToolSchema(
         filters: filtersSchema
             .nullable()
             .describe(
-                'Filters to apply to the query. Filtered fields must exist in the selected explore.',
+                'Filters to apply to the query. Filtered fields must exist in the selected explore or should be referenced from the custom metrics. If they are custom metrics, place them in the metrics filter group.',
```

Comment:
"If they are custom metrics, place them in the metrics filter group."

This is something I'd expect to be defined in the system prompt, we have `metrics/dimensions` split in multiple different schemas and the same idea applies

Reply:
Testing all tools by removing the very last sentence - I've also updated the other tool args to just be 

```
.describe(
                'Filters to apply to the query. Filtered fields must exist in the selected explore or should be referenced from the custom metrics.',
            ),
```